Main.swift

```diff
@@ -0,0 +1 @@
+print("Hello, swft!")
```

Comment:
```suggestion
print("Hello, swift!")
```
Swiftへの愛（`i`）が足りてないね。